index.html

```diff
@@ -0,0 +1,252 @@
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+      <section class="card-works">
+        <img src="./img/Snapshoot-Portfolio-2.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Tonic</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+      <section class="card-works-1">
+        <img src="./img/Snapshoot-Portfolio-3.svg" alt="Snapshoot Portfolio" />
+        <h1 class="project-title">Multi-Post Stories</h1>
+        <ul class="frame-2">
+          <li class="client">Canopy</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="role">Back End Dev</li>
+          <li><img src="./img/Counter.svg" alt="" /></li>
+          <li class="year-label">2015</li>
+        </ul>
+        <p class="primary-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="tags">
+          <li class="tag">html</li>
+          <li class="tag">css</li>
+          <li class="tag">javascript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </section>
+    </div>
+    <footer class="base-about-myself">
+      <section class="top">
+        <h1>
+          About <br />
+          Myself
+        </h1>
+        <p class="top-presentation">
+          I'm a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
+          me.
+        </p>
+        <p class="top-connect">Let's connect</p>
+        <div class="top-social-media">
+          <img src="./img/github.svg" alt="" class="flex-item" /><img
+            src="./img/Linkedin-icon.svg"
+            alt=""
+            class="flex-item"
+          /><img src="./img/angellist.svg" alt="" class="flex-item" /><img
+            src="./img/twitter.svg"
+            alt=""
+            class="flex-item"
+          /><img src="./img/medium.svg" alt="" class="flex-item" />
+        </div>
```

Comment:
When we are dealing with multiple links like in this instance, for better practices and semantics, we should group them in a ul tag and each list item inside the ul element should be as a li 😁 Also, please be sure that each icon is wrapped with an `a` tag 🔥🤓

Reply:
I did